### DISCUSSION TITLE
What about the Intel code?

### DISCUSSION THREAD

**Discussion by Lalarian** [vosen/ZLUDA#96] [Q&A]
So it first was developed for Intel GPUs. Then for AMD.
What about the older Intel capabilities?
Did you remove them?
Or are they stil in there, too?
What Intel GPUs would it have been compatible to?
It would be so great, if Intel IGPs could run CUDA .. with obviousy a huge mass of addtional people then being able to utilize it.

**Comment by kamieniarzadam** [comment]
On releases page of the project its clearly written, that new version removes support for Intel graphics, but you can still use older version.
I have seen somewhere, that only pre Xe cards are supported.
